### PR TITLE
added link from quickref to ref of PIO assembly instructions

### PR DESCRIPTION
**Pull request by NitiKaur**

### CODE REVIEW

**Comment by dpgeorge** [comment]
Please add a short example for using PIO, like this:
```
Example using PIO to blink an LED at 1Hz:

from machine import Pin
import rp2


@rp2.asm_pio(set_init=rp2.PIO.OUT_LOW)
def blink_1hz():
    # Cycles: 1 + 7 + 32 * (30 + 1) = 1000
    set(pins, 1)
    set(x, 31)                  [6]
    label("delay_high")
    nop()                       [29]
    jmp(x_dec, "delay_high")

    # Cycles: 1 + 7 + 32 * (30 + 1) = 1000
    set(pins, 0)
    set(x, 31)                  [6]
    label("delay_low")
    nop()                       [29]
    jmp(x_dec, "delay_low")


# Create and start a StateMachine with blink_1hz, outputting on Pin(25)
sm = rp2.StateMachine(0, blink_1hz, freq=2000, set_base=Pin(25))
sm.active(1)
```

File: docs/library/sys.rst [hunk 1]
```diff
@@ -115,6 +115,13 @@ Constants
    If you need to check whether your program runs on MicroPython (vs other
    Python implementation), use `sys.implementation` instead.
 
+.. data:: settrace
+
+   Requires custom MicroPython compilation as it is ,ostly not present in
```
Comment (mattytrentini) [inline docs/library/sys.rst:120]: Suggest: "as it is typically not present in"

File: docs/library/sys.rst [hunk 2]
```diff
@@ -115,6 +115,13 @@ Constants
...
+   pre-built  firmware (due to its affecting performance). The relevant build
```
Comment (mattytrentini) [inline docs/library/sys.rst:121]: Please remove the extra space between pre-built and firmware. 
Suggest: "(due to the negative effect on performance)"

**Comment by dpgeorge** [comment]
Thanks for this, merged in e538d8a5a6d4c643b1bff41107062511b2e7f65a and b851950c26e185d46c16b993d310506992fbdb89

**Pull request closed without merging**